.github/workflows/merge.yml

```
@@ -19,7 +19,7 @@ jobs:
     strategy:
       matrix:
         os: [windows-2019, macos-latest, ubuntu-latest]
-        node-version: [14, 16]
+        node-version: [14, 16, 18]
```

I think this should be:

```suggestion
        node-version: [14, 18]
```

(the earliest and latest node versions supported)